webextensions/api/devtools.json

```diff
@@ -6,11 +6,9 @@
           "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/devtools",
           "support": {
             "chrome": {
-              "version_added": true
-            },
-            "edge": {
-              "version_added": "79"
+              "version_added": "≤54"
```

Comment:
@queengooborg I cannot find any sub-feature listing version 54 for Chrome. Also, there still seem to be a number of 'true' statements against features for Chrome (e.g. $0 line 123) - I thought the idea was to eliminate all of these?

Reply:
I think this was caused by a typo when setting the `panels` subfeature, accidentally copying the version number from Firefox.  Fixing now!

Reply:
Ah, I see what it is now -- it's because subfeatures for Opera have version numbers that imply support in earlier versions of Chromium:

<img width="1027" alt="Screenshot 2024-07-11 at 07 41 20" src="https://github.com/mdn/browser-compat-data/assets/5179191/8e9e76e7-e763-4500-aced-5c78a8a09156">

I opted to mirror the data in reverse as I thought that was most likely accurate, but now that I'm looking at some of the data points, I see that there are some discrepancies between the Chrome and Opera data.  In `webextensions.api.devtools.panels.themeName`, Chrome is set to 59, but Opera is set to 41.

What do you think we should do in these cases?

Reply:
@queengooborg I assume, like you, I've got no particular information to use to judge the reliability of this older compatibility information. The options are:

1.  to assume that we can't be any more reliable than saying "58 or earlier" based on the creation date of the original BCD file, or
2. that all support version information for chromium-based browsers is valid, e.g., vse 41 in this case. 

Given that version 59 is now over seven years old, I'm not sure either choice will have any practical ramifications. Given that we are keeping historic Edge data and assume that all other historic data (versions prior to the creation of the BCD file) is correct, I would suggest using known information where we have it.

Reply:
Since these odd discrepancies in the data will need to be researched further to determine what the accurate data is, I decided to revert those changes and de-mirror Opera instead.  Now, all of the changes in this PR are strictly either for "≤58" as implied from the commit that added these features to BCD, or from subfeatures that can easily be verified!